### PR TITLE
protect: add new 30/500 protection

### PR DESCRIPTION
**Pull request by MusikAnimal**
This is subject to misuse, so including a confirmation popup indicating it is (currently) strictly for Arbitration purposes on qualifying subjects.

### CODE REVIEW

**Comment by MusikAnimal** [comment]
@atlight Do you care to review this? Only thing that worries me is the protection weight stuff. I tested it out. E.g. the page was semi'd, I tried extendend confirmed and it counts as an increase. Meanwhile if it's already extended confirmed, requesting PC or semi counts as a decrease.

File: modules/twinkleprotect.js [hunk 1]
```diff
@@ -718,6 +736,7 @@ Twinkle.protect.protectionWeight = {
 	sysop: 30,
 	templateeditor: 20,
 	flaggedrevs_review: 15,  // Pending Changes level 2 protection alone
+	extendedconfirmed: 12,
```
Comment (atlight) [inline modules/twinkleprotect.js]: Shouldn't be 12. This is for semiprotected + PC1 (see comment above).  Maybe it is time to increase sysop to 40, TE to 30, then put 30/500 at 20. Unless PC2 alone (15) and semiprotected + PC2 (17) should really be considered "higher" than 30/500?

File: modules/twinkleprotect.js [hunk 2]
```diff
@@ -915,13 +941,23 @@ Twinkle.protect.callback.changePreset = function twinkleprotectCallbackChangePre
...
+			isArbEnforcement = item.edit === 'extendedconfirmed';
+
 		if (mw.config.get('wgArticleId')) {
 			if (item.edit) {
 				form.editmodify.checked = true;
 				Twinkle.protect.formevents.editmodify({ target: form.editmodify });
 				form.editlevel.value = item.edit;
 				Twinkle.protect.formevents.editlevel({ target: form.editlevel });
+
+				// NOT: Currently extendedconfirmed protection is only for Arbitration enforcement,
```
Comment (atlight) [inline modules/twinkleprotect.js]: `NOTE`?

File: modules/twinkleprotect.js [hunk 3]
```diff
@@ -1021,6 +1068,14 @@ Twinkle.protect.callback.evaluate = function twinkleprotectCallbackEvaluate(e) {
 		case 'protect':
 			// protect the page
 
+			if (form.editlevel.value === 'extendedconfirmed') {
+				if (!confirm('Extended confirmed protection is only for Arbitration enforcement. ' +
+						'Please confirm that the subject of this page is a qualifying topic (see WP:30/500)'
```
Comment (atlight) [inline modules/twinkleprotect.js]: Full stop needed at the end of the message. And you can use && to merge the two if statements into one and avoid crazily deep indent levels.

File: modules/twinkleprotect.js [hunk 4]
```diff
@@ -1129,6 +1184,15 @@ Twinkle.protect.callback.evaluate = function twinkleprotectCallbackEvaluate(e) {
 
 		case 'request':
 			// file request at RPP
+
+			if (form.category.value === 'pp-30-500') {
+				if (!confirm('Extended confirmed protection is only for Arbitration enforcement. ' +
+						'Please confirm that the subject of this page is a qualifying topic (see WP:30/500)'
```
Comment (atlight) [inline modules/twinkleprotect.js]: Same as above.

**Comment by atlight** [comment]
Duly reviewed.

**Comment by MusikAnimal** [comment]
@atlight those weight changes worked. Also cleaned up the confirm popups. Thanks for the help, I'll deploy once I get a final OK

**Comment by MusikAnimal** [comment]
Merging!

**Pull request merged**